tools/proto-js-plugin/src/main/java/io/spine/js/generate/parse/ParsingOfObjects.java

```diff
@@ -60,57 +58,34 @@
  *         message stored in a file.
  * </ol>
  */
-public final class JsonParsersWriter {
+public final class ParsingOfObjects extends FileSetEnhancement {
```

Comment:
This class name is strange. From the name, it seems like the object describes the process of parsing.
Please think of a better name.